app/controllers/middleware_server_controller.rb

```diff
@@ -135,10 +136,15 @@ def add_jdbc_driver
       :driver_minor_version => params["minorVersion"]
     }
 
-    run_server_operation(STANDALONE_SERVER_OPERATIONS.fetch(:middleware_add_jdbc_driver), selected_server)
-    render :json => {
-      :status => :success, :msg => _("JDBC Driver \"%s\" has been installed on this server.") % params["driverName"]
-    }
+    op_trigger = run_server_operation(STANDALONE_SERVER_OPERATIONS.fetch(:middleware_add_jdbc_driver), selected_server)
+    if op_trigger
+      if params[:id].nil?
+        add_flash(_("JDBC Driver \"%s\" has been installed on selected servers.") % params["driverName"], :success)
```

Comment:
Please, use named parameters for format expansion under gettext. http://manageiq.org/docs/guides/i18n